src/options.js

```diff
@@ -38,6 +39,8 @@ export interface ArsOptions {
   columns: ArsColumn[];
   // Existing selections
   picked?: ID | ID[];
+  // What utility should Ars use for network requests?
+  request: (url: string, success: *, error: *) => XMLHttpRequest;
```

Comment:
I'm a fan of [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), but I'm not sure if this would mean we'd need to configure for polyfills since that would exclude IE.

Reply:
Yeah, I didn't want to introduce the Promise + Fetch requirement, but we could revisit that. It's becoming less and less of an issue.

This work hasn't released yet. We could totally do this.

Reply:
Ahh right. The other issue is that I abort requests for data when they're no longer needed, fetch won't natively support that until this becomes standard:

https://developer.mozilla.org/en-US/docs/Web/API/AbortController

Support is lame :(, but I'd want to pull it in to cancel prior requests.

Reply:
ah, then let's skip fetch then. Is aborting requests something that library users will have to worry about if they provide their own request?

Reply:
With the current contract, as long as they return an XMLHttpRequest, it's fine.